src/main/resources/styles/lightmode.css

```
@@ -0,0 +1,414 @@
+}
+
+/* =========================
+   LOADING OVERLAY (LIGHT)
+   ========================= */
+
+.loading-overlay {
+    -fx-background-color: rgba(255, 255, 255, 0.6);
+}
+
+.loading-text {
+    -fx-text-fill: #222222;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)